src/xcode/ENA/ENA/Source/Services/OnBehalfCheckinSubmission/OnBehalfCheckinSubmissionError.swift

```diff
@@ -34,14 +34,25 @@ enum OnBehalfCheckinSubmissionError: LocalizedError, Equatable {
 			default:
 				return "\(AppStrings.OnBehalfCheckinSubmission.Error.tryAgain) (\(failure))"
 			}
-		case .submissionTANError(let failure):
+		case .registrationTokenError(let failure):
 			switch failure {
-			case .serverError(let statusCode) where (400...409).contains(statusCode):
+			case .receivedResourceError(let registrationTokenError):
+				switch registrationTokenError {
+				case .regTokenNotExist :
```

Comment:
I don't see that this error case had an explicit handling in the old code.
Can you point me to the place where "registrationTokenError.errorDescription" was returned in the old code?

Reply:
you mean the regTokenDid not exist ?

Reply:
if yes then check the comment where i mentioned you